libs/ledger-live-common/src/families/stacks/__tests__/sync.integration.test.ts

```diff
@@ -1,7 +1,9 @@
 import "../../../__tests__/test-helpers/setup";
-import { fetchFullTxs } from "../bridge/utils/api";
+// import { fetchFullTxs } from "../bridge/utils/api";
```

Comment:
can be removed

Reply:
done in this commit 5f029be7a28183d5d2a83a3d19e7f52b149ef185